build.sbt

```
@@ -124,33 +83,29 @@ lazy val `netty-ssl` = project
 
 lazy val server = project
   .in(file("modules/server"))
-  .dependsOn(common % "compile->compile;test->test")
-  .dependsOn(`internal-core` % "compile->compile;test->test")
-  .dependsOn(`internal-monix` % "test->test")
-  .dependsOn(`internal-fs2` % "test->test")
-  .dependsOn(channel % "test->test")
-  .dependsOn(testing % "test->test")
+  .dependsOn(`rpc-service`)
   .settings(moduleName := "mu-rpc-server")
   .settings(serverSettings)
 
 /////////////////////
 //// HEALTHCHECK ////
 /////////////////////
 
-lazy val `health-check-unary` = project
-  .in(file("modules/health-check-unary"))
-  .dependsOn(channel)
+lazy val `health-check` = project
+  .in(file("modules/health-check"))
+  .dependsOn(`rpc-service`)
+  .dependsOn(fs2)
+  .dependsOn(monix)
```

```suggestion
  .dependsOn(fs2 % "optional->compile")
  .dependsOn(monix % "optional->compile")
```

I've confirmed locally that this generates an `optional` dependency in the pom:

```
        <dependency>
            <groupId>io.higherkindness</groupId>
            <artifactId>mu-rpc-fs2_2.12</artifactId>
            <version>0.21.3+52-523d4b3e+20200504-0852-SNAPSHOT</version>
            <optional>true</optional>
        </dependency>
```

---

Wonderful 👍 Thanks!